src/vs/workbench/contrib/positronVariables/browser/components/variablesInstance.tsx

```diff
@@ -513,6 +513,17 @@ export const VariablesInstance = (props: VariablesInstanceProps) => {
 		}
 	};
 
+	// workaround for web disabling scrolling on the window to prevent URL navigation
+	useEffect(() => {
```

Comment:
Curious why we're using `useEffect` here instead of adding an `onWheel` to the div? I think the other places we've added this fix used `onWheel`.